dev/load.go

```diff
@@ -109,7 +109,7 @@ func runConstantRequestsPerSecond(url string, jsonBytes []byte) {
 	ticker := time.NewTicker(_requestInterval)
 	done := make(chan bool)
 
-	c := make(chan os.Signal)
+	c := make(chan os.Signal, 1)
```

Comment:
How does this affect the `load.go` app?

Reply:
This should have no effect, but it was necessary to satisfy `go vet` on go 1.17

Reply:
Got it.